container-images/kolla/base/start.sh

```diff
@@ -9,6 +9,9 @@ sudo -E kolla_set_configs
 CMD=$(cat /run_command)
 ARGS=""
 
+# Install custom CA certificates
+sudo kolla_update_cacerts
```

Comment:
not all users are in kolla group and can run this via sudo

Reply:
well `sudo -E kolla_set_configs` above is also good, so we can expect this to work

Reply:
it's `kolla_copy_cacerts`  in kolla, should we try to keep it consistent?

Reply:
we could, thought we are now disconnected from kolla.

Reply:
yeah but likely to hit similar issues in both, with similar fixes, so good to stay as consistent as possible I expect